### PR TITLE
adding ip, updating layout of channel header, adding test coverage

### PR DESCRIPTION
**Pull request by kaplanmaxe**
Implements #138 & v2 refactor of #154 

Adding IP to the UI, layout changes in `ChannelsHeader`, starting test coverage for wallet.

Using ipify.org which is a popular, public API without authentication / limits.

### CODE REVIEW

File: src/actions/wallet.js [hunk 1]
```diff
@@ -97,6 +97,16 @@ class ActionsWallet {
...
   }
+
+  async getIP() {
+    try {
+      const request = await fetch('https://api.ipify.org?format=json');
+      const response = await request.json();
+      this._store.IP = response.ip;
+    } catch (e) {
+      throw new Error('Error fetching IP');
+    }
```
Comment (tanx) [inline src/actions/wallet.js]: Seems like the app would crash if IP detection via the api fails. Either we log `e` to the console or handle it some other way.

File: src/actions/wallet.js [hunk 2]
```diff
@@ -97,6 +97,16 @@ class ActionsWallet {
         this.t2342 = setTimeout(() => this.getNewAddress(), RETRY_DELAY);
       });
   }
+
+  async getIP() {
+    try {
+      const request = await fetch('https://api.ipify.org?format=json');
+      const response = await request.json();
+      this._store.IP = response.ip;
```
Comment (tanx) [inline src/actions/wallet.js]: Please rename to `store.ipAddress`.

File: src/actions/wallet.js [hunk 3]
```diff
@@ -97,6 +97,16 @@ class ActionsWallet {
         this.t2342 = setTimeout(() => this.getNewAddress(), RETRY_DELAY);
       });
   }
+
+  async getIP() {
```
Comment (tanx) [inline src/actions/wallet.js]: Please rename to `getIPAddress`.

**Comment by kaplanmaxe** [comment]
@tanx updated per review

**Comment by tanx** [comment]
Thanks @kaplanmaxe. Right now it's fine to detect the IP. We should deactivate this feature by default thought to protect the user's privacy. Created a ticket here: https://github.com/lightninglabs/lightning-app/issues/197

**Pull request merged**